### PR TITLE
Update user-tokens.md

### LINKED ISSUE #5360: Make sure we consistency refer to api keys and PATs appropriately
### Contributions

- [x] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

To reduce confusion, refer to the old keys as user API keys or API keys and the new account-scoped PATs as that or PATs for short. 

See [internal convo](https://dbt-labs.slack.com/archives/C019QLR6EJE/p1713887216140899?thread_ts=1713498266.415409&cid=C019QLR6EJE) for context

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens and other pages mentioning tokens

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by phoenixmjay**
Consistently refer to User API keys and Personal Access Tokens (PAT) appropriately

Closes https://github.com/dbt-labs/docs.getdbt.com/issues/5360

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 1]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
...
+Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using Service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
 
 PATs inherit the permissions of the user that created them. For example, if a developer-licensed user with Project Admin role access to specific projects creates a PAT, the token will get the Project Admin role with access to the same projects as the user. These tokens are also account-specific, so if a user has access to more than one dbt Cloud account with the same email address, they need to create a unique PAT for each one of these accounts. 
 
-### Migrate from user API keys to personal access tokens
+### Migrate from user API keys to Personal Access Tokens
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]: @Millacy Heads up that we use [sentence case](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Titles) in title for a more conversational tone.
```suggestion
### Migrate from user API keys to personal access tokens
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 2]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
...
 
-The current API key is located under **Personal Settings → API Key.** 
+The current user API key is located under **Personal Settings → API Key.** 
 
 Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.
 
 :::
 
-## Account-scoped personal access tokens
+## Account-scoped Personal Access Tokens
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]: @Millacy Heads up that we use [sentence case](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Titles) in titles for a more conversational tone.
```suggestion
## Account-scoped personal access tokens
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 3]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
 
 :::note Announcement
 
-The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). We recommend rotating your existing user tokens with PATs. There are no deprecation plans for user API tokens at this time; we will give ample notice when that timeline has been determined. 
+[User API keys](#user-tokens) are being replaced by [account-scoped Personal Access Tokens(PATs)](#account-scoped-personal-access-tokens).
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]: We should only capitalize products (dbt Explorer, dbt Cloud, but not features like column-level lineage), Acronyms and proper nouns. I will add this to the style guide so it's better communicated!
```suggestion
[User API keys](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). 
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 4]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
...
 
 :::
 
-Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
+Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using Service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]:
```suggestion
Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 5]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
...
-### Migrate from user API keys to personal access tokens
+### Migrate from user API keys to Personal Access Tokens
 
 The migration to PATs is critical if you are using user API keys today. The current API key is located under **Personal Settings → API Key**.
 
  There are a few things to understand if you are using a user API key today: 
 
-* Personal access tokens are more secure. 
-    * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped personal access tokens.
+* Personal Access Tokens are more secure.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]:
```suggestion
* Personal access tokens are more secure. 
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 6]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
...
+### Migrate from user API keys to Personal Access Tokens
 
 The migration to PATs is critical if you are using user API keys today. The current API key is located under **Personal Settings → API Key**.
 
  There are a few things to understand if you are using a user API key today: 
 
-* Personal access tokens are more secure. 
-    * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped personal access tokens.
+* Personal Access Tokens are more secure. 
+    * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped Personal Access Tokens.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]:
```suggestion
    * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped personal access tokens.
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 7]
```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
...
 
-## Account-scoped personal access tokens
+## Account-scoped Personal Access Tokens
 
 :::info New 
 
-On Feb 7, 2024, we introduced a new type of token for individual users called personal access tokens. Note that these differ from [Service Tokens or API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 
+On Feb 7, 2024, we introduced a new type of token for individual users called Personal Access Tokens. Note that these differ from [Service Tokens or user API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). 
+
+Before this release, User API keys were the only way to access dbt Cloud API on behalf of the user.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]: This sentence feels very linked to the previous paragraph. Is there a reason you separated it from the paragraph?
```suggestion
Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. 
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 8]
```diff
@@ -50,26 +56,30 @@ If you’re using any undocumented and unsupported API endpoints, please note th
...
 :::
 
-### Using the new personal access tokens
+### Using the new Personal Access Tokens
+
+Are you using a user API key today to access dbt Cloud APIs in any of your workflows? 
+
+If not, you don’t have any action to take! 
 
-Are you using a user API key today to access dbt Cloud APIs in any of your workflows? If not, you don’t have any action to take. If you are using a user API key, please follow the instructions below.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md:57]: Nice catch!

**Review by runleonarun — CHANGES_REQUESTED** [review]
Thank you @Millacy for making the docs more consistent! I undid some capitalization changes, and will get our style guide updated to reflect my suggestions.

File: website/docs/docs/dbt-cloud-apis/user-tokens.md (hunk 7 above)
Comment (phoenixmjay) [reply website/docs/docs/dbt-cloud-apis/user-tokens.md]: Added a break for readability, the large block felt difficult to parse:
Screenshot 2024-05-29 at 9 57 21 PM

Thoughts on keeping the first paragraph break below if i remove that 2nd paragraph break? Or would it be better to keep it as it was before?
Screenshot 2024-05-29 at 10 02 21 PM

**Review by matthewshaver — APPROVED** [review]

**Comment by runleonarun** [comment]
@matthewshaver and @mirnawong1 Update captured by Phoenix [here](https://github.com/dbt-labs/docs.getdbt.com/issues/5360#issuecomment-2142731405) The UI still says "API Tokens" 

cc @saimaddali

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 9]
```diff
@@ -8,11 +8,13 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
 
 :::note Announcement
 
-The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). We recommend rotating your existing user tokens with PATs. There are no deprecation plans for user API tokens at this time; we will give ample notice when that timeline has been determined. 
+[User API keys](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). 
+
+We recommend rotating your existing user API keys with PATs. There are no deprecation plans for user API keys at this time; we will give ample notice when that timeline has been determined.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]:
```suggestion
We recommend rotating your existing user API keys with PATs. At this time, there are no deprecation plans for user API keys; we will give ample notice when that timeline has been determined. 
```

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 10]
```diff
@@ -52,26 +57,30 @@ If you’re using any undocumented and unsupported API endpoints, please note th
...
 
-### Using the new personal access tokens
+### Using the new Personal Access Tokens
+
+Are you using a user API key today to access dbt Cloud APIs in any of your workflows? 
+
+If not, you don’t have any action to take! 
 
-Are you using a user API key today to access dbt Cloud APIs in any of your workflows? If not, you don’t have any action to take. If you are using a user API key, please follow the instructions below. 
+If you are using a user API key, please follow the instructions below:
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]:
```suggestion
If you are using a user API key, please read the following instructions:
```

**Comment by matthewshaver** [comment]
Going to convert to `draft` and have added the `blocked by dev` label here as well.

**Comment by matthewshaver** [comment]
Closing as we deprecated the old keys and since they no longer work the page has been updated.

**Pull request closed without merging**